build-parent/pom.xml

```diff
@@ -513,6 +513,7 @@
       </activation>
       <properties>
         <jacoco.skip>false</jacoco.skip>
+        <sonar.exclusion>**/RunCucumberTest.java</sonar.exclusion>
```

Comment:
It may have more sense to add the exclusion into affected modules rather than parent, as the value is specific to those tests only. WDYT?